client/app/components/packages/rwcds-form/with-action-no-action.hbs

```diff
@@ -65,7 +65,7 @@
               </Q.Label>
 
               <p class="q-help">
-                If Department of Building (DOB) approvals were granted that support the future No-Action Scenario assumed in the prior section, please indicate that in this summary, and if available, attach the DOB approved plans to this document.
+                If Department of Building (DOB) approvals were granted that support the future No-Action Scenario assumed in the prior section, please indicate that in this summary, and if available, attach the DOB approved plans to this document. Refer to the CEQR Technical Manual, Chapter 2, (pages 2-5, 2-6, &amp; 2-7)”.
```

Comment:
Should there be a doublequote here?

Reply:
Nope.  It's gonna be gone in this next push